src/public/css/styles.css

```diff
@@ -35,26 +34,30 @@ div.user {
 }
 
 input[type="submit"], button {
-  transition: .2s;
-  padding: 0.5em 1.5em;
+  transition: background-color .9s;
+  padding: 0.6em 2.5em;
   color: white;
   border: 0;
-  border-radius: 2.5px;
+  border-radius: 3px;
   outline: none;
   cursor: pointer;
+  font-family: roboto;
```

Comment:
You don't import Roboto so this won't work unless you've got the font installed on your computer.